fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/StaticWikiConfigLoader.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.wiki.webservice.data.config.auto;
+
+import com.yahoo.bard.webservice.data.config.names.TableName;
+import com.yahoo.bard.webservice.data.time.DefaultTimeGrain;
+import com.yahoo.bard.webservice.data.time.TimeGrain;
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by kevin on 3/7/2017.
+ */
+public class StaticWikiConfigLoader implements ConfigLoader {
+
```

Comment:
Implements Supplier<List<DataSourceConfiguration>> 
(method name would change to get() )